app/src/main/java/ch/epfl/polybazaar/filllisting/FillListingActivity.java

```diff
@@ -250,18 +241,24 @@ private boolean fillFieldsIfEdit() {
         imageManager.retrieveAllImages(listStringImage, listImageID, listingID);
         titleSelector.setText(listing.getTitle());
         descriptionSelector.setText(listing.getDescription());
-        freeSwitch.setChecked(listing.getPrice().equals("0.0"));
         priceSelector.setText(listing.getPrice());
-        Category editedCategory = new NodeCategory(listing.getCategory());
-        Category root = RootCategoryFactory.getDependency();
-        traversingCategory = root.getSubCategoryContaining(editedCategory);
-        categorySelector.setSelection(root.indexOf(traversingCategory)+1);
         lat = listing.getLatitude();
         lng = listing.getLongitude();
         if (lat != NOLAT && lng != NOLNG) {
             addMP.setText(R.string.change_MP);
-            meetingPointStatus.setText(R.string.MP_ok);
         }
+
+        /**
+         * FOR TESTING PURPOSES ONLY:
```

Comment:
Again I don't understand this comment

Reply:
I left the category stuff in there, so as to not have to rewrite all tests right now. Ultimately these lines will be removed, once the new category selection has been implemented.